RULES.md

```diff
@@ -0,0 +1,94 @@
+# Traffic rules
+
+## TLDR
+We need to:
+
+Allow independent groups in the content project to work at their own pace on parts of the project that don’t influence each other.
+Introduce enter and exit criteria for roles with elevated privileges so the hierarchy can self-organize.
+Define a framework for organizations to join the project and assume responsibilities.
+Make sure that the project stays coherent and individual parts contribute to its value, and the project doesn’t become a disgusting spaghetti soup.
+
+
+## Decomposition
+
+In order to keep the project scalable, we divide the project into separate areas, and assign groups of users that can be granted commit access with the intention to develop content in respective areas.
+Those areas are going to be segmented by
+
+- Products (RHEL, Ubuntu, …) - make it easy to develop content of a product that doesn’t influence the rest of the build system.
```

Comment:
How's this managed hierarchically? Wouldn't Ubuntu contributors necessarily have access to all Ubuntu-product-specific profiles? Or is this additive rather than absolute (e.g., any of the Ubuntu contributors/maintainers _plus_ the CIS-specific ones in the example below)?